# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../jamin/compile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Some trivial compile fixes.

  - Rene Rebe <rene@exactcode.de>

--- ./src/scenes.c.orig	2004-11-06 17:41:36.909229194 +0000
+++ ./src/scenes.c	2004-11-06 17:42:32.852324030 +0000
@@ -60,9 +60,6 @@
                          *LED_yellow = NULL, *LED_red = NULL;
 
 
-void set_EQ_curve_values ();
-
-
 /*  Initialize all scene related structures and get the widget addresses.  */
 
 void bind_scenes ()
@@ -210,7 +207,7 @@
 
                         s_crossfade_to_state (&scene_state[i], -1.0f);
 
-                        set_EQ_curve_values ();
+                        set_EQ_curve_values (0, 0.0);
 
                         s_history_add_state (scene_state[i]);
 
